generator/generator_config.textproto

```diff
@@ -381,6 +381,14 @@ service {
   retryable_status_codes: ["kUnavailable"]
 }
 
+service {
```

Comment:
Is this service GA?  The release notes say it is preview:

https://cloud.google.com/build/docs/release-notes#January_23_2023

And I do not see anything changing that to GA.  If it is not GA we should consider adding the "experimental" stuff.

Reply:
Yeah, you're right. I followed the `datacatalog/lineage` precedent.

I updated the parent issue to include a step for promoting the library to GA, and marked this PR as only doing part of the work.

Also, FWIW, I verified that I could actually call the API:

```
INFO: Running command line: google/cloud/cloudbuild/quickstart/quickstart foo
name: "projects/cloud-cpp-testing-resources/locations/us-central1/connections/dbolduc-test"
create_time {
  seconds: 1679501900
  nanos: 702999604
}
update_time {
  seconds: 1679501989
  nanos: 53807970
}
github_config {
  authorizer_credential {
    oauth_token_secret_version: "projects/936212892354/secrets/dbolduc-test-github-oauthtoken-aa4368/versions/1"
    username: "dbolduc"
  }
  app_installation_id: 35540553
}
installation_state {
  stage: COMPLETE
}
etag: "KnoukQtcRX1OzB4lPx7fRZIgOREMpwpGRIFGX_WfNPE"
```